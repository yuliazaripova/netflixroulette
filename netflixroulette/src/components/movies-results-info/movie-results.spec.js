import * as React from 'react';

import MoviesResultsInfo from './movies-results-info';

describe("MoviesResultsInfo component", () => {
  it("should render MoviesResultsInfo component", () => {
    const component = shallow(<MoviesResultsInfo qt={1}/>);
    expect(component).toMatchSnapshot();
  })
})