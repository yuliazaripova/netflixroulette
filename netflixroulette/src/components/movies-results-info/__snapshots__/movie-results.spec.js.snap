// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesResultsInfo component should render MoviesResultsInfo component 1`] = `
<Fragment>
  <span
    className="movie-results-info_qt"
  >
    1
     movies found
  </span>
  <RadioToolbar
    items={
      Array [
        Object {
          "id": "release",
          "isChecked": true,
          "value": "release date",
        },
        Object {
          "id": "rating",
        },
      ]
    }
    name="movie-results-info"
  />
</Fragment>
`;
